Junk4Dollars-app/Auctions/AuctionListTableViewDataSource.swift

```diff
@@ -10,11 +10,16 @@ public class AuctionListTableViewDataSource: NSObject, UITableViewDataSource {
 
     public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = UITableViewCell()
-        cell.textLabel?.text = auctions[indexPath.row].title
+        cell.textLabel?.text = auctionFor(indexPath: indexPath).title
         return cell
     }
     
     public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return auctions.count
     }
+
+    public func auctionFor(indexPath: IndexPath) -> Auction {
+        let auction = auctions[indexPath.row]
```

Comment:
As is, this can be inlined
```suggestion
        return auctions[indexPath.row]
```